Assets/HoloToolkit/Input/Prefabs/MixedRealityCamera.prefab

```diff
@@ -1,667 +0,0 @@
-%YAML 1.1
-%TAG !u! tag:unity3d.com,2011:
```

Comment:
If we're getting rid of the MixedRealityCamera.prefab would this PR get a breaking change label?

Reply:
It probably should. I'll re-add it for now, but it should be removed sooner rather than later.